src/app/loadout-drawer/loadout-utils.ts

```diff
@@ -644,7 +647,7 @@ export function getUninstancedLoadoutItem(
   storeId: string | undefined,
 ) {
   // This is mostly for subclasses - it finds all matching items by hash and then picks the one that's on the desired character
-  const candidates = potentialUninstancedLoadoutItemsByHash(allItems).get(hash) ?? [];
+  const candidates = uninstancedItemsByHash(allItems).get(hash) ?? [];
```

Comment:
Subclasses are `instanced` technically but we call them uninstanced because we match by hash. This regresses subclass lookup.

Reply:
Oh also emblems... not sure if the instanced check in the lookup table buys us much